app/workers/application_worker.rb

```diff
@@ -1,21 +1,12 @@
-require "prx_access"
+require "prx/api"
 
 class ApplicationWorker
-  include PrxAccess
   include Shoryuken::Worker
 
-  attr_accessor :message
-
   def self.prefix_name(name)
     [prefix, application, name].join("_")
   end
 
-  def self.announce_queues(model, actions)
```

Comment:
Oh wow, there was still `Announce` stuff around!

Reply:
not much! but when I ran across this I figured I should cleanse it